setup.py

```diff
@@ -37,10 +54,12 @@
         "pandas>=1.0.1,!=1.5.0",
         "scikit-image>=0.17",
         "scikit-learn>=1.0",
-        "scipy>=1.9",
+        "scipy>=1.4,<1.11.0",
         "statsmodels>=0.11",
         "tables>=3.7.0",
-        "torch<=1.12",
+        "timm",
+        "torch>=2.0.0",
```

Comment:
is the older transformer for re-ID working with torch > 2? cc @yeshaokai please test and confirm.